### ISSUE TITLE
255 Character limit for input arguments

### ISSUE THREAD

**Issue by willirath** [meom-group/CDFTOOLS#13]
Currently, there are many [256-character limits](https://github.com/meom-group/CDFTOOLS/search?q=LEN%3D256&type=Code). It would be nice to expand this to allow for (much) more than this.

An example: I have `section_*.dat`-files I use with `cdf_xtrac_brokenline` to extract data along all of the WOCE sections. Naming the section files `section_WOCEA1.dat,section_WOCEA2.dat,...` etc. I get a `cldum` about 1300 characters.

The limit used to be 80 characters and has already been expanded to 256 characters in: 6652eb459705653500c9e30e2445387b07b287c1

How far could we go with this?  Would it be problematic to allow for `LEN=2048` or even more?

**Comment by molines** [comment]
I have no particular concern about raising the limit to 2048 characters. I have not tried already if fortran support this, but I guess yes. 

But there also might be an issue with the linux maximum length for a command line, I am pretty sure that is well above, however. 

The nicer solution will be to allow for 1 file with all sections inside the file ( just a concatenation of separate section files).  Just need to write the parser for this kind of file. Should not be too complicated as the number of points defining the section is already coded into the bloc for a section... Nice exersize, I will think about it !

**Comment by willirath** [comment]
> I have no particular concern about raising the limit to 2048 characters. I have not tried already if fortran support this, but I guess yes.
> 
> But there also might be an issue with the linux maximum length for a command line, I am pretty sure that is well above, however.

I'll try it on our machines and let you know.

> The nicer solution will be to allow for 1 file with all sections inside the file ( just a concatenation of separate section files). Just need to write the parser for this kind of file. Should not be too complicated as the number of points defining the section is already coded into the bloc for a section... Nice exersize, I will think about it !

Before you do this: The current version has some nice effects, too. I like being able to do
```bash
cdf_xtrac_brokenline ... -f `ls -1 section_*.dat | paste -sd,`
```
and adding sections by just committing a new file to the list of available sections.

**Comment by molines** [comment]
Yes I agree, in any case, if I code the possibility for a single file ( multi section) I will keep also the original possibility (multi files) ..[ and the question of cldum remains the same :)...
